src/main/java/io/confluent/idesidecar/restapi/application/SidecarInfo.java

```diff
@@ -142,12 +155,13 @@ public String osArch() {
 
   @Override
   public String toString() {
-    return "OS: %s %s (%s); VS Code %s, extension version %s".formatted(
+    return "OS: %s %s (%s); VS Code %s, extension version %s, callback URI scheme %s".formatted(
```

Comment:
Open to better wording here - maybe replacing "VS Code" with the URI scheme altogether?